fe/fe-core/src/main/java/com/starrocks/load/DeleteHandler.java

```diff
@@ -603,7 +603,7 @@ private boolean checkDeleteV2(OlapTable table, List<Partition> partitions, List<
                     updatePredicate(binaryPredicate, column, 1);
                 } catch (AnalysisException e) {
                     // ErrorReport.reportDdlException(ErrorCode.ERR_INVALID_VALUE, value);
-                    throw new DdlException("Invalid column value[" + value + "] for column " + columnName);
+                    throw new DdlException("Invalid value for column " + columnName + ": " + e.getMessage());
```

Comment:
suggest to tell user that the "value for column is out-of-range,  legel values  must range in (${MIN}, ${MAX})"